internal/config/config.go

```diff
@@ -37,6 +37,9 @@ const (
 	DefaultNamespace                = "default"
 	VulnerabilityType               = "vuln_mgmt"
 	AssetInventoryType              = "asset_inventory"
+	ProviderAWS                     = "aws"
+	ProviderAzure                   = "azure"
+	ProviderGCP                     = "gcp"
```

Comment:
Just for sake of "consistency", maybe we should have this values inside of a provider file, like benchmark has?

https://github.com/elastic/cloudbeat/blob/main/internal/config/benchmark.go

Reply:
That's great, definitely want to keep magic strings in one place if possible. Will do!